router/terraform/cloudfront.tf

```diff
@@ -69,6 +69,16 @@ resource "aws_cloudfront_distribution" "wellcomecollection_org" {
         ]
       }
     }
+
```

Comment:
I've left this out purposefully, to get this PR in, then will attach the lambdas to our default cache behaviour.